webapp/channels/src/packages/mattermost-redux/src/actions/integrations.ts

```diff
@@ -380,6 +421,28 @@ export function regenOAuthAppSecret(appId: string) {
     });
 }
 
+export function deleteOutgoingOAuthConnection(id: string): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        try {
+            await Client4.deleteOutgoingOAuthConnection(id);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+
+            dispatch(logError(error));
+            return {error};
+        }
+
+        dispatch(batchActions([
+            {
+                type: IntegrationTypes.DELETED_OUTGOING_OAUTH_CONNECTION,
+                data: {id},
+            },
+        ]));
```

Comment:
Nit, this doesn't need `batchActions` since it's just a single action
```suggestion
        dispatch({
            type: IntegrationTypes.DELETED_OUTGOING_OAUTH_CONNECTION,
            data: {id},
        });
```